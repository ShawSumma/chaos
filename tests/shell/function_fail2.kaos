any f = f(1,2)
symbol_table

void def f()
    bool f = true
    return f
end

f(1,2)
f()
