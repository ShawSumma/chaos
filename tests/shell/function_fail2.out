Chaos Error:
Module: ~/chaos/__interactive__.kaos
Line: 1
Undefined function: f
Absorbed by Interactive Shell
[start] =>
[end]
Chaos Error:
Module: ~/chaos/__interactive__.kaos
Line: 8
Incorrect argument count for function: f
Absorbed by Interactive Shell
Chaos Error:
Module: ~/chaos/__interactive__.kaos
Line: 1
Illegal variable type: Boolean for function: f
Absorbed by Interactive Shell

You have chosen the order!
